### DISCUSSION TITLE
Hint on how to restore

### DISCUSSION THREAD

**Discussion by cMurschall** [prodrigestivill/docker-postgres-backup-local#109] [Q&A] [answered]
I'm really embarrassed to ask, but I inherited a project using your image and I wonder on how to restore a backup.
It's a totally noob question and I'm sorry, but I don't have much experience on how to service a postgres database and perhaps you have a hint for me. 

I tried the accepted answer https://stackoverflow.com/a/29913462/4634044 but with no luck. 
```
cat backup.sql | docker exec -i image psql --username=myusername
```
failes with 
`psql: FATAL:  database "myusername" does not exist`

Perhaps you could share how you restore your backups?

**Comment by cMurschall** [comment, marked as answer]
I figured out the problem: I was missing a `-d databaseName` parameter:
```
cat backup.sql | docker exec -i image psql --username=myusername --dbname=myDataBaseName
```

**Comment by ghost** [comment]
It would still be useful if there is a line added to the documentation describing how to do this.
